build.sbt

```diff
@@ -23,7 +23,7 @@ developers := List(
 )
 
 scalaVersion := "2.13.10"
```

Comment:
```suggestion
scalaVersion := "2.13.11"
```

Reply:
Using `2.13.11` as the main version fails checks for missing dependencies.

Reply:
I updated scoverage which should fix the problem. It seems it was using scapegoat itself creating a circular dependency but it's scoverage. :)